charts/dsv-injector/Chart.yaml

```diff
@@ -9,7 +9,7 @@ keywords:
   - secrets
   - vault
 type: application
-version: 0.2.0
+version: 0.2.1
 appVersion: latest
 maintainers:
   - name: Adam Migus
```

Comment:
```suggestion
```

@tylerezimmerman  is this what you wanted? If so accept suggested change. I think you meant to remove Adam from maintainer list.